torch/jit/__init__.py

```diff
@@ -1941,6 +1967,14 @@ def linear(x):
     """
     return False
 
+
+def is_tracing():
+    """
+    Returns ``True`` in tracing (if a function is called during the tracing of
+    code with ``torch.jit.trace``) and ``False`` otherwise.
+    """
+    return torch._C._is_tracing
```

Comment:
Returns the function, and not the bool `torch._C._is_tracing()`. `_script_if_tracing` doesn't work on 1.6.0 but is fixed on master.

Reply:
Fixed [here](https://github.com/pytorch/pytorch/pull/42486) for reference.